frontend/app/routes/$lang+/_protected+/letters+/index.tsx

```diff
@@ -122,9 +124,11 @@ export default function LettersIndex() {
       </ul>
 
       <div className="my-6 flex flex-wrap items-center gap-3">
-        <ButtonLink id="back-button" to="/home">
-          {t('letters:index.button.back')}
-        </ButtonLink>
+        {userOrigin && (
```

Comment:
If `userOrigin` does not exist, we will be rendering an empty `<div>`. Include the `<div className=my-6 flex ...">` inside the `userOrigin` condition to prevent that.

Reply:
FIxed: 142c0903fce0ba9db8090c28e8a8062f77a9ff3f